Electrify.AdminUi.UnitTests/Services/ConnectedClientsServiceTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Electrify.AdminUi.Services;
+using Electrify.AdminUi.Services.Abstractions;
+using Electrify.Models;
+using Electrify.Server.ApiClient.Abstraction;
+using Electrify.Server.ApiClient.Contracts;
+using FluentAssertions;
+using NSubstitute;
+using Xunit;
+
+namespace Electrify.AdminUi.UnitTests.Services
+{
+    public class ConnectedClientsServiceTests
+    {
+        private readonly IElectrifyApiClient _mockApiClient;
```

Comment:
i think* these should be the implementations not interfaces?

Reply:
may be wrong, but VS always corrects it when i write unit tests

Reply:
nah but its not the SUT so I dont think it matters idk.